addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties

```diff
@@ -1,8 +1,35 @@
 # This file defines the default (English) variants of all of the internationalised messages of the plugin
 
-spiderajax.api.action.scan = Runs the spider against the given URL and/or context, optionally, spidering everything in scope. The parameter 'contextName' can be used to constrain the scan to a Context, the option 'in scope' is ignored if a context was also specified. The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
-spiderajax.api.action.scanAsUser = Runs the spider from the perspective of a User, obtained using the given context name and user name. The parameter 'url' allows to specify the starting point for the spider, otherwise it's used an existing URL from the context (if any). The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
-
+# Api Views
+spiderajax.api.view.fullResults = Shows the full spidered content detected by the AjaxSpider.
+spiderajax.api.view.numberOfResults = Shows the number of detected resources.
+spiderajax.api.view.optionBrowserId = Shows the tipe of browser being used for the crawl, this has been set at launch time (The values should be: Chrome, Chrome-headless, Firefox, Firefox-headless, HtmlUnit, PhantomJS, Safari).
+spiderajax.api.view.optionClickDefaultElems = Shows the configured value on clicking actions on defaul html tag elements such as "a" "button" "input", all asociated with some action or links on the page.
```

Comment:
This needs tweaks, maybe `Shows the configured value for "Click Default Elements Only", HTML elements such as "a" "button" "input".`?